app/views/shortened_links/_form.html.erb

```diff
@@ -0,0 +1,70 @@
+<div class="live-modal"
+     data-controller="shortened-link-form"
+     data-action="keydown->shortened-link-form#handleKeyDown"
+     data-shortened-link-form-link-id="<%= shortened_link.id %>"
+>
+  <div class="live-modal-backdrop" data-action="click->shortened-link-form#closeModal">
+    <div class="live-modal-content" data-action="click->shortened-link-form#trapClick">
+      <div class="live-modal-close" data-action="click->shortened-link-form#closeModal">&times;</div>
+      <% if form_action == :edit %>
```

Comment:
I thought this needed to have the `@` like `@form_action` 🤔

Reply:
I sent it as a local 😄